src/view/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { App, MarkdownPostProcessorContext, MarkdownRenderChild } from "obsidian";
+import { getAPI, isPluginEnabled } from "obsidian-dataview";
+
+import ObsidianFeedsPlugin from "../main";
+import getSettings, { Settings } from "./settings";
+import renderError from "./errors";
+
+export default class ObsidianFeeds extends MarkdownRenderChild {
+  private settings: Settings;
+
+  constructor(
+    public plugin: ObsidianFeedsPlugin,
+    public src: string,
+    public containerEl: HTMLElement,
+    public app: App,
+    public ctx: MarkdownPostProcessorContext,
+  ) {
+    super(containerEl);
+    this.settings = getSettings(src, app, containerEl, ctx);
+  }
+
+  async onload() {
+    const hasDataView = isPluginEnabled(this.app);
+    if (!hasDataView) {
+      const error =
+        "Dataview plugin is not installed. Please install community plugin.";
```

Comment:
[🔧](https://github.com/erikthedeveloper/code-review-emoji-guide#:~:text=:wrench:)
```suggestion
        "Dataview plugin is not installed. Please install it from Community plugins.";
```